### ISSUE TITLE
No Image Inversion

### ISSUE THREAD

**Issue by gr4nny** [krassowski/Anki-Night-Mode#3]
OSX 10.11.2
Anki Version 2.0.33
Qt 4.8.2 PyQt 4.9.4

![anki - james-1](https://cloud.githubusercontent.com/assets/8114687/12401604/8799ed88-bdf5-11e5-8519-1a8674a1acd4.jpg)

Also, I have css specifying text as Navy. On the android night mode it automatically reformat this to some shade of yellow. It seems that your plugin only overrides the default text color.

**Comment by krassowski** [comment]
Hi,
1. Do you have "Invert images" enabled in menu? (View → Night Mode → Invert Images)
2. Yes, currently there is no full colour inversion of texts as it is implemented in AnkiDroid.

**Comment by gr4nny** [comment]
Yes. This is occurring with local images and linked images (src="http://...). 

I just edited the cards and changed it to gray, so it was no big deal.

**Comment by gr4nny** [comment]
Also, when I am in full screen mode, edit a card > save change
I get a blank, dark screen. If I alt-tab twice it redisplays correctly. Again, no big deal, just an FYI.

**Comment by krassowski** [comment]
Ok, so last from basic questions: Do you have the last version of add-on (1.1.2) installed?

If updating add-on doesn't help, I would recommend you to update Qt to 4.8.6 version (if it's possible). I don't know what exactly causes the bug (at least currently) but it might be connected with qt webkit version, which was updated in last minor releases of 4.8 branch.

I will investigate further after my finals, now I'm a little bit to busy.

**Comment by gr4nny** [comment]
I have 1.1.2.
I do not know how to update Qt separately. I will have to look into that.
Good skill on your finals. No worries and thanks for your work.

**Comment by altre** [comment]
Same problem here with the same version numbers.
I played around with the css in the card directly and using "filter" does not work, so it does seem to be some webkit problem.
My python interpreter gives different versions for qt and pyqt than anki does.

>>> from PyQt4.QtCore import *
>>> QT_VERSION_STR
'4.8.7'
>>> PYQT_VERSION_STR
'4.11.4'

**Comment by krassowski** [comment]
Ok, finally I'm able to replicate this bug (on Windows). Here Qt and PyQt versions are independent of those installed in the system, since Anki provides own copies of *pyqt dlls. These dlls were last time modified in 2013, so it is the next premise, that they are just obsolete, and (probably) old version of webkit ignores 'filter' property. I tried to replace them in-situ to never versions, but this doesn't work so easy - Anki stops working after that. Maybe compiling Anki with newer PyQt will solve the problem.

**Comment by altre** [comment]
Mh, just cloned anki and ran it with ./tools/build_ui.sh and ./runanki. Now it uses the newer qt versions: Version 2.0.33
Qt 4.8.7 PyQt 4.11.4
But the problem remains.

**Comment by krassowski** [comment]
On Windows or other OS?

**Comment by altre** [comment]
Sorry, same as gr4nny: OSX 10.11.2

**Comment by krassowski** [comment]
@gr4nny @altre do you still use Anki? Could you test the addon in the Anki 2.1 beta 16?
